packages/components/components/elvis-datepicker/CHANGELOG.json

```diff
@@ -2,6 +2,18 @@
   "$schema": "../../changelogSchema.json",
   "name": "elvis-datepicker",
   "content": [
+    {
+      "date": "01.10.24",
+      "version": "9.1.0",
+      "changelog": [
+        {
+          "type": "new_feature",
+          "changes": [
+            "Added automatic support for switching aria-labels based on the global lang attribute on the html-element."
```

Comment:
```suggestion
            "Added automatic support for switching aria-labels based on the global <code>lang</code> attribute on the <code>html</code>-element."
```

Reply:
Nit: Consider adding "updated internal dependencies" (or something like that) as the `@elvia/elvis-typography` dependency was updated.